pkgs/servers/web-apps/wordpress/update.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p common-updater-scripts jq wp4nix
+
+set -eu -o pipefail
+
+version=$(curl --globoff "https://api.wordpress.org/core/version-check/1.7/" | jq -r '.offers[0].version')
+update-source-version wordpress $version
+
+# cd /tmp
```

Comment:
Are these comments still relevant?

Reply:
You can use them to find out the officially packaged themes and plugins as they change at least every year with a newly packaged theme.